include/arch/x86/linker.ld

```diff
@@ -137,6 +137,19 @@ SECTIONS
 
 #include <linker/common-ram.ld>
 
+#ifdef CONFIG_MMU
+       /*
+        * Page Tables are located here if MMU is enabled.
+	*/
+	SECTION_DATA_PROLOGUE(mmu_page_tables, (OPTIONAL),)
```

Comment:
Does this really need to be in its own binary section or can we place this in the data section? For example we put the generated IDT directly in rodata.